src/main/java/com/webperside/courseerpbackend/services/student/StudentServiceImpl.java

```diff
@@ -15,6 +24,13 @@
 public class StudentServiceImpl implements StudentService {
```

Comment:
as I said in the task description, we'll use business services here. 
Because if we do all logic in these services for student logic, after a while it will be more complex and hard to test and hard to find bugs

please, start to investigate AuthBusinessServiceImpl first. You will see there we are using services like that in the business service. for example UserService, CourseService, and so on. 

tips: You will create StudentBusinessService and put this addStudent logic there. 
and remove the injections from the student service which added by you

Reply:
StudentBusinessService also will be in the services/student package ?

Reply:
@gadirovakhadija yes